While you're at it, can you fix https://github.com/duckdb/duckdb/pull/21282#issuecomment-4111743576 ?
This spams every build with warnings 😅 

```
[247/913] Building CXX object src/storage/table/CMakeFiles/duckdb_storage_table.dir/ub_duckdb_storage_table.cpp.o
In file included from /Users/thijs/DuckDBLabs/duckdb_iceberg/build/debug/src/storage/table/ub_duckdb_storage_table.cpp:16:
/Users/thijs/DuckDBLabs/duckdb_iceberg/duckdb/src/storage/table/row_group_reorderer.cpp:178:51: warning: missing field 'leading_null_group_offset' initializer [-Wmissing-field-initializers]
  178 |                                 return {new_row_offset, pruned_row_group_count};
      |                                                                               ^
/Users/thijs/DuckDBLabs/duckdb_iceberg/duckdb/src/storage/table/row_group_reorderer.cpp:197:48: warning: missing field 'leading_null_group_offset' initializer [-Wmissing-field-initializers]
  197 |         return {new_row_offset, pruned_row_group_count};
      |                                                       ^
/Users/thijs/DuckDBLabs/duckdb_iceberg/duckdb/src/storage/table/row_group_reorderer.cpp:178:51: warning: missing field 'leading_null_group_offset' initializer [-Wmissing-field-initializers]
  178 |                                 return {new_row_offset, pruned_row_group_count};
      |                                                                               ^
/Users/thijs/DuckDBLabs/duckdb_iceberg/duckdb/src/storage/table/row_group_reorderer.cpp:300:17: note: in instantiation of function template specialization 'duckdb::(anonymous namespace)::FindOffsetPrunableChunks<std::__map_iterator<std::__tree_iterator<std::__value_type<duckdb::Value, duckdb::(anonymous namespace)::RowGroupOffsetEntry>, std::__tree_node<std::__value_type<duckdb::Value, duckdb::(anonymous namespace)::RowGroupOffsetEntry>, void *> *, long>>, std::__map_iterator<std::__tree_iterator<std::__value_type<duckdb::Value, duckdb::(anonymous namespace)::RowGroupOffsetEntry>, std::__tree_node<std::__value_type<duckdb::Value, duckdb::(anonymous namespace)::RowGroupOffsetEntry>, void *> *, long>>>' requested here
  300 |                 auto result = FindOffsetPrunableChunks(ordered_row_groups.begin(), ordered_row_groups.end(), order_by,
      |                               ^
/Users/thijs/DuckDBLabs/duckdb_iceberg/duckdb/src/storage/table/row_group_reorderer.cpp:197:48: warning: missing field 'leading_null_group_offset' initializer [-Wmissing-field-initializers]
  197 |         return {new_row_offset, pruned_row_group_count};
      |                                                       ^
/Users/thijs/DuckDBLabs/duckdb_iceberg/duckdb/src/storage/table/row_group_reorderer.cpp:178:51: warning: missing field 'leading_null_group_offset' initializer [-Wmissing-field-initializers]
  178 |                                 return {new_row_offset, pruned_row_group_count};
      |                                                                               ^
/Users/thijs/DuckDBLabs/duckdb_iceberg/duckdb/src/storage/table/row_group_reorderer.cpp:306:17: note: in instantiation of function template specialization 'duckdb::(anonymous namespace)::FindOffsetPrunableChunks<std::reverse_iterator<std::__map_iterator<std::__tree_iterator<std::__value_type<duckdb::Value, duckdb::(anonymous namespace)::RowGroupOffsetEntry>, std::__tree_node<std::__value_type<duckdb::Value, duckdb::(anonymous namespace)::RowGroupOffsetEntry>, void *> *, long>>>, std::reverse_iterator<std::__map_iterator<std::__tree_iterator<std::__value_type<duckdb::Value, duckdb::(anonymous namespace)::RowGroupOffsetEntry>, std::__tree_node<std::__value_type<duckdb::Value, duckdb::(anonymous namespace)::RowGroupOffsetEntry>, void *> *, long>>>>' requested here
  306 |                 auto result = FindOffsetPrunableChunks(ordered_row_groups.rbegin(), ordered_row_groups.rend(), order_by,
      |                               ^
/Users/thijs/DuckDBLabs/duckdb_iceberg/duckdb/src/storage/table/row_group_reorderer.cpp:197:48: warning: missing field 'leading_null_group_offset' initializer [-Wmissing-field-initializers]
  197 |         return {new_row_offset, pruned_row_group_count};
      |                                                       ^
```

--- a/src/storage/table/row_group_reorderer.cpp
+++ b/src/storage/table/row_group_reorderer.cpp
@@ -175,7 +175,7 @@ OffsetPruningResult FindOffsetPrunableChunks(It it, End end, const OrderByStatis
 			if (pruned_stats->CanHaveNull()) {
 				// Offset pruning can only remove a contiguous prefix of row groups. Once the first unresolved
 				// row group might still contribute NULLs to the ordered prefix, we cannot prune past it.
-				return {new_row_offset, pruned_row_group_count};
+				return {new_row_offset, pruned_row_group_count, 0};
 			}
 			// This row group has exactly row_group.count valid values. We can exclude those.
 			pruned_row_group_count++;
@@ -194,7 +194,7 @@ OffsetPruningResult FindOffsetPrunableChunks(It it, End end, const OrderByStatis
 		}
 	}
 
-	return {new_row_offset, pruned_row_group_count};
+	return {new_row_offset, pruned_row_group_count, 0};
 }
 
 } // namespace

